src/browser/index.js

```diff
@@ -1,5 +1,5 @@
 import './vdiff.js';
 
-export { assert, aTimeout, defineCE, expect, html, nextFrame, oneEvent, waitUntil } from '@open-wc/testing';
+export { assert, aTimeout, defineCE, expect, html, nextFrame, oneEvent, oneDefaultPreventedEvent, waitUntil } from '@open-wc/testing';
```

Comment:
Surprised this didn't fail alphabetical linting rule...

```suggestion
export { assert, aTimeout, defineCE, expect, html, nextFrame, oneDefaultPreventedEvent, oneEvent, waitUntil } from '@open-wc/testing';
```

Reply:
From what I can tell, that linting rule only cares about `import` sorting and not `export`s.